lib/afc/streams.js

```diff
@@ -20,21 +20,23 @@ class AfcReadableFileStream extends stream.Readable {
     }
 
     if (_.isEmpty(data)) {
-      await this._destroy();
+      this.destroy();
```

Comment:
how about 

> Use end() instead of destroy if data should flush before close, or wait for the 'drain' event before destroying the stream

Reply:
I can check method

Reply:
That method doesn't exist in Readable Streams